application/language/spanish/index.html

```diff
@@ -1,10 +1,11 @@
-﻿<html>
+<!DOCTYPE html>
+<html>
 <head>
-	<title>403 Prohibido</title>
+	<title>403 Forbidden</title>
```

Comment:
Let's keep the language-specific 403 pages as it is.

Reply:
See other comment on russion